### PR TITLE
[Artifacts] Add "Register Artifact" button and dialog

### PR DESCRIPTION
**Pull request by IvanVoznyi**

### CODE REVIEW

File: src/common/Input/input.scss [hunk 1]
```diff
@@ -25,6 +25,11 @@
     transform: translate(0, -50%);
   }
 
+  &-default {
+    border: $primaryBorder;
+    border-radius: 2px;
+  }
```
Comment (mavdryk) [inline src/common/Input/input.scss:31]: why do you need a new style for inputs?

File: src/common/PopUpDialog/PopUpDialog.js [hunk 2]
```diff
@@ -1,58 +1,29 @@
...
-const PopUpDialog = ({
-  actionBtnText,
-  children,
-  closeError,
-  closePopUp,
-  handleSuccess,
-  headerText,
-  message
-}) => {
+const PopUpDialog = ({ headerText, children, closePopUp }) => {
```
Comment (mavdryk) [inline src/common/PopUpDialog/PopUpDialog.js:8]: sort props

File: src/common/Input/input.scss (hunk 1 above)
Comment (mavdryk) [reply src/common/Input/input.scss:31]: I believe all inputs should have these styles. And you don't need this new class

File: src/common/PopUpDialog/popUpDialog.scss [hunk 3]
```diff
@@ -19,13 +20,23 @@
...
+      line-height: 28px;
+    }
+
+    &-close {
+      cursor: pointer;
+    }
   }
 
-  &__btn {
+  &__buttons {
```
[the remark this answers is not in the document]
Comment (IvanVoznyi) [reply src/common/PopUpDialog/popUpDialog.scss:39]: I needed to rename this class to "buttons" because class "btn" broken down layout.

**Pull request closed without merging**